theia-java-docker/Dockerfile

```diff
@@ -1,7 +1,7 @@
 
 FROM ibmcom/ibmjava:8-sdk as java-base
 
-FROM ibmcom/ibmnode:8
+FROM node:8
```

Comment:
I think we should roll-back this line and if needed open a separate issue to replace obsolete image ibmcom/ibmnode .

Reply:
Got it: We'll do that shortly after this PR.